content/cli/commands/produce.md

```diff
@@ -221,13 +226,71 @@ partitions using [`fluvio partition list`].
 %copy first-line%
 ```bash
 $ fluvio partition list
- TOPIC          PARTITION  LEADER  REPLICAS  RESOLUTION  HW  LEO  LSR  FOLLOWER OFFSETS
- multi-no-keys  0          0       []        Online      2   2    0    []
- multi-no-keys  1          0       []        Online      2   2    0    []
- multi-no-keys  2          0       []        Online      2   2    0    []
- multi-no-keys  3          0       []        Online      2   2    0    []
- multi-no-keys  4          0       []        Online      2   2    0    []
+ TOPIC          PARTITION  LEADER  REPLICAS  RESOLUTION  SIZE   HW  LEO  LRS  FOLLOWER OFFSETS 
+ multi-no-keys  0          0       []        Online      120 B  2   2    0    [] 
+ multi-no-keys  1          0       []        Online      121 B  2   2    0    [] 
+ multi-no-keys  2          0       []        Online      124 B  2   2    0    [] 
+ multi-no-keys  3          0       []        Online      126 B  2   2    0    [] 
+ multi-no-keys  4          0       []        Online      127 B  2   2    0    [] 
 ```
 
 Notice how the high watermark (HW) and log-end-offset (LEO) tell us that there are
 exactly 2 records in each partition. Our ten records have been evenly distributed!
+
+## Example 4: Producing using a compression algorithm (GZIP)
+
+Fluvio support different types of compression algorithms to send records. 
+Compression, in general, improves throughput in exchange of some CPU cost to compress/uncompress the data.
+
+Let's try to use `gzip` algorithm in the CLI.
```

Comment:
Should rename `gzip` topic to `compressed` to remove confusion.
Also `none` to `uncompressed` or `text`.